git/homework.py

```diff
@@ -15,23 +15,32 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     If @first and @second has same value should return True
     In another case should return False
     """
-    pass
+    if first == second:
+        return True
+    else:
+        return False
 
```

Comment:
redundant statement "if". Operator "==" will already return True or False.
Make the refactoring.